src/mainboard/pcengines/apu2/mainboard.c

```diff
@@ -141,25 +142,43 @@ static void pirq_setup(void)
 /* Wrapper to enable GPIO/UART devices under menuconfig. Revisit
  * once configuration file format for SPI flash storage is complete.
  */
-#define SIO_PORT 0x2e
+#define SIO_PORT 0x2E
 
 static void config_gpio_mux(void)
 {
 	struct device *uart, *gpio;
 
-	uart = dev_find_slot_pnp(SIO_PORT, NCT5104D_SP3);
-	gpio = dev_find_slot_pnp(SIO_PORT, NCT5104D_GPIO0);
-	if (uart)
-		uart->enabled = CONFIG_APU2_PINMUX_UART_C;
-	if (gpio)
-		gpio->enabled = CONFIG_APU2_PINMUX_GPIO0;
-
-	uart = dev_find_slot_pnp(SIO_PORT, NCT5104D_SP4);
-	gpio = dev_find_slot_pnp(SIO_PORT, NCT5104D_GPIO1);
-	if (uart)
-		uart->enabled = CONFIG_APU2_PINMUX_UART_D;
-	if (gpio)
-		gpio->enabled = CONFIG_APU2_PINMUX_GPIO1;
+	if (check_uartc()) {
+		printk(BIOS_INFO, "UARTC enabled\n");
+
+		uart = dev_find_slot_pnp(SIO_PORT, NCT5104D_SP3);
+		if (uart) uart->enabled = 1;
```

Comment:
I know this code is ported from legacy, but please fix it to be compliant with coreboot coding standard. `if` and expression should go to separate lines.